dataflow/snippets/read_kafka.py

```diff
@@ -42,16 +41,16 @@ def _add_argparse_args(parser: argparse.ArgumentParser) -> None:
         (
             pipeline
             # Read messages from an Apache Kafka topic.
-            | ReadFromKafka(
-                consumer_config={"bootstrap.servers": options.bootstrap_server},
-                topics=[options.topic],
-                with_metadata=False,
-                max_num_records=5,
-                start_read_time=0,
+            | beam.managed.Read(
+                beam.managed.KAFKA,
+                config={
+                  "bootstrap_servers": options.bootstrap_server,
+                  "topic": options.topic,
+                  "data_format": "RAW",
+                  "auto_offset_reset_config": "earliest",
+                  "max_read_time_seconds": 5  # For testing, avoid in production.
```

Comment:
nit: can you clarify this comment. I looked it up in documentation and it looks like this _option_ is intended for testing or other non-production scenarios. This comment as is currently could be read as the 5 seconds is for testing.

Reply:
That makes sense, thanks. Clarified the comment.